test/modelManagers/usermodel.test.js

```diff
@@ -0,0 +1,17 @@
+const { expect } = require('chai');
+const { LIMITHELPER, OFFSETHELPER } = require('../../constants');
+const User = require('../../lib/modelManagers/usermodel');
+
+describe('Unit test usermodel allusers', () => {
```

Comment:
My expectation here is that you would add a given number of users to the database and expect that the number of users you got back is equal to the number of users you created. I would also expect that you would assert that users profile is added to users record returned.

Reply:
i have added this.